package.json

```diff
@@ -245,5 +244,8 @@
 		"webpack-bundle-analyzer": "^3.6.1",
 		"webpack-cli": "^3.3.11",
 		"webpack-merge": "^4.2.2"
+	},
+	"resolutions": {
+		"@babel/preset-env": "7.9.0"
```

Comment:
based on this: https://stackoverflow.com/questions/60780664/could-not-find-plugin-proposal-numeric-separator